### DISCUSSION TITLE
List of followed tags and possibility to add/remove tags from lists

### DISCUSSION THREAD

**Discussion by Aristarch** [elk-zone/elk#1951] [Ideas]
In the original Mastodon Web UI, there is a route to list the #hashtags I am following: /followed_tags
I would really be delighted to have this list in Elk available, too!
And a second, related issue another guy suggested in a discussion I had a while ago: It would be nice being able to add and remove followed hashtags to lists the same way as followed users. But I'm not sure if this is technically supported by the Mastodon backend since it isn't an available feature in the original Web UI as well.

**Comment by Shinigami92** [comment]
I think it is supported and it is of a high possibility that the other guy was me 🙋

Theoretically I also need this list of following hashtags for #1940
So now we have already two use cases for different features to store the following hashtags in some kind of storage (e.g. localstorage or idb) but then we would also need to sync this sometimes in the background 🤔
